### PR TITLE
*: update dependencies

### PR DESCRIPTION
**Pull request by paulfantom**
As in title.

Side question: Should I add an automatic `jb update && make generate` as a CI cronjob? It would create a PR only when something in `manifests/` was changed.

### CODE REVIEW

File: manifests/kube-state-metrics-prometheusRule.yaml [hunk 1]
```diff
@@ -40,3 +40,26 @@ spec:
       for: 15m
       labels:
         severity: critical
+    - alert: KubeStateMetricsShardingMismatch
```
Comment (lilic) [inline manifests/kube-state-metrics-prometheusRule.yaml:43]: The metrics for this are only available in master right now, so this won't do anything. Are we sure we are pinning to latest release?

Comment (paulfantom) [reply manifests/kube-state-metrics-prometheusRule.yaml:43]: We were pinning to `release-1.9` for ksm deployment but to `master` for mixin. Now it should be fixed. Great catch!

**Review by lilic — APPROVED** [review]
lgtm

File: manifests/kube-state-metrics-prometheusRule.yaml (hunk 1 above)
Comment (lilic) [reply manifests/kube-state-metrics-prometheusRule.yaml]: Yeah we didn't used to have any diff between master and other branches, but we recently merged these cool metrics and alerts. :)

**Comment by paulfantom** [comment]
Let's put this on hold until KSM v2-rc.0 is merged.

**Comment by lilic** [comment]
@paulfantom any reason for that? I will get to it tomorrow or on Friday but I don't see a reason why we can't merge this now.

**Comment by paulfantom** [comment]
This way we won't introduce unnecessary changes to alerting rules which we almost immediately revert.

**Comment by brancz** [comment]
I don't really understand what we would be reverting. Can you elaborate? ksm 2.0 seems strictly additive to these changes, so in fact it would just blow this PR up further making it harder to review.

**Comment by lilic** [comment]
I opened https://github.com/prometheus-operator/kube-prometheus/pull/1041 but there are no unnecessary changes to alerting rules we would revert. So not sure what you meant there, can you elaborate?

**Comment by paulfantom** [comment]
There are just cosmetic changes like swapping `message` with `description` in alert annotations. Feel free to merge it if you think it is not that big of a deal, I don't want to block on such minor things.

**Comment by paulfantom** [comment]
Updated PR to include latest kubernetes-mixins (with KSM v2 metrics) and prometheus mixin.

**Comment by paulfantom** [comment]
I am merging this as it was already approved and brings in necessary changes needed for ksm v2

**Pull request merged**